src-docs/src/components/guide_section/guide_section.tsx

```diff
@@ -133,7 +133,7 @@ export const GuideSection: FunctionComponent<GuideSection> = ({
 
   const renderPlaygroundToggle = () => {
     const isPlaygroundUnsupported =
-      typeof window !== 'undefined' && typeof document !== 'undefined';
+      typeof window === 'undefined' || typeof document === 'undefined';
 
     if (!isPlaygroundUnsupported && !!playground) {
```

Comment:
I feel like a `isPlaygroundSupported` would be clearer than `isPlaygroundUnsupported`, but I'm not going to block over something that trivial

Reply:
Agree! Let's see what Josh says. Indeed, that minor issue could bring missread when defining that variable.